src/adapters/mc/ssh/LinuxSshServerSecurityBaseline_DeployIfNotExists.json.tmpl

```diff
@@ -14,8 +14,8 @@
                 "name": "LinuxSshServerSecurityBaseline",
                 "version": "1.0.0",
                 "contentType": "Custom",
-                "contentUri": "https://github.com/Azure/azure-osconfig/releases/download/test_policy_package/LinuxSshServerSecurityBaseline.zip",
-                "contentHash": "1ADA673657D30BC454919EB74AC05785ED946FC948AD64CFFFF436BC229567C6",
+                "contentUri": "@URI@",
+                "contentHash": "@HASH@",
```

Comment:
Please do not change these test policy definitions. 
Also please do not delete or change any existing GitHub releases.

The nrp-build.yml is used mainly for the CI automation for the MC tests, which need only the policy packages, they do not need any policy definitions.

The prod build of the policy packages is in process of being moved to OneBranch.